var/spack/repos/builtin/packages/scotch/package.py

```diff
@@ -104,6 +107,12 @@ def libs(self):
 
         return scotchlibs + zlibs
 
+    @when("+fismahigh")
+    def patch(self):
+        for path in ["ci/analysis.sh", "src/misc/scotch_5.1.12_openmpi.spec"]:
+            if os.path.exists(path):
+                os.remove(path)
```

Comment:
I don't understand, what is the reason to remove these?

Reply:
These reflect requirements from the security team within NOAA/NWS. Basically for a given package, they give us a list of required changes. It's usually files that have URLs or IP addresses, wget or curl commands, etc.